README.md

```diff
@@ -1,70 +1,176 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
```

Comment:
- [ ] On your reservations list:

![image](https://github.com/huzaifakhan0308/full-stack-capstone-front-end/assets/57421823/967acc38-e2cc-44fb-8ced-15664d6c2528)

We need the name of the room/hotel to be shown instead  of just the id.  Kindly implement this so we can meet this [requirement ](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md#project-requirements)and improve the UX. 👍 

- [ ] On that same page, the reservations feature a "Cancel reservation" button that has no functionality.  Kindly either implement the functionality or remove the button so the UX is improved. 💪